docs/testcases.md

```diff
@@ -194,13 +201,13 @@ Click on another failure line in the failed step.
 
 **Expected**: The log should scroll to that failure line.
 
-Click on 'show successful steps'.
+Click on 'show successful steps'. (DELETE ME? I can't find this on a failing mochitest log)
```

Comment:
I'm not sure if these should be removed. Maybe a different job type would show this, or maybe this was removed a long time ago and no longer applies...

Reply:
Yeah, this was removed since Taskcluster jobs don't have "steps" like BuildBot jobs did.